content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -9,6 +9,8 @@ weight: 10
 
 <!-- overview -->
 
+CronJobs was promoted to `batch/v1` in v1.21, for previous versions of kubernetes, please refer to appropriate documentation sets on the website.
```

Comment:
nit: Not sure about the statement to view prior docs? The last 4 versions are available on the site.
CronJob is promoted to `batch/v1` in Kubernetes v1.21.

Reply:
As these changes will merge into the primary branch after the v1.21 release, I think the statement is correct.

How about:
```suggestion
CronJobs was promoted to general availability in Kubernetes v1.21. If you are using an older version of
Kubernetes, please refer to the documentation for the version of Kubernetes that you are using,
so that you see accurate information. Older Kubernetes versions do not support the `batch/v1` CronJob
API.
```
?

Reply:
sgtm, updated.